### ISSUE TITLE
10x MT genes and Neutrophils 

### ISSUE THREAD

**Issue by flde** [LungCellAtlas/HLCA#4]
Dear all, 

Many thanks for providing the great HLCA! Maybe you could help me with two quick questions about the 10x data. 

1) There are no mitochondrial (MT) genes present. Where they filtered out or removed was the genome removed during the read mapping? 
2) In the 10x I can't find Neutrophils, only in the plate based assay. However, it seems that the samples where processed identically prior sequencing. Is there a reason why they are absent in the 10x?

All the best and many thanks for your help, 
Florian

**Comment by LisaSikkema** [comment]
Hi Florian,

Happy to hear you're trying out the HLCA!
Are you using the latest version of the HLCA (posted on cellxgene only a few days ago)? It should have MT genes, I am able to see them in the cellxgene visualisation... Based on what do you conclude they are not there?

As to neutrophils: these are generally not well captured in 10X data (see e.g. https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/tutorials/neutrophils). As the HLCA core consists only of 10X data, and the rest of the data was annotated through label transfer from the HLCA core, they are not annotated in the entire HLCA. 

In the `original_ann_nonharmonized` you'll find two studies have annotated neutrophils, but one of those (the Nawijn one) later recognised that this was a misannotation.

**Comment by flde** [comment]
Hi Lisa, 

Thank you very much for the explanation! I just realized that the MT was missing from the Stanford HLCA not yours. Sorry about that. 

Very much looking forward to the final version! 

All the very best, 
Florian

**Comment by flde** [comment]
Hello Alexander, 

I thought the low gene count is due to RNase and inhibitors stored in neutrophils are released during lysis in GEM - and not a biological feature? 

Hence, most studies sort out neutrophils from inflamed tissue to ensure yield and overall sample quality. But I remember that there are 10x protocols to prep granulocytes (somewhat a sc-seq blind spot). 

The reason I asked was because Travaglini et al annotated lung neutrophils but they were not present in the 10x data. I figured now that they do have captured some with plate based techniques https://hlca.ds.czbiohub.org/. I just posted to the wrong HLCA GitHub (awkward). 

Best wishes, 
Florian
